river/command/rule.zig

```diff
@@ -85,6 +87,13 @@ pub fn ruleAdd(_: *Seat, args: []const [:0]const u8, _: *?[]const u8) Error!void
                 .value = tag,
             });
         },
+        .output => {
+            try server.config.output_rules.add(.{
+                .app_id_glob = app_id_glob,
+                .title_glob = title_glob,
+                .value = try util.gpa.dupe(u8, result.args[0]),
```

Comment:
The duped string leaks when adding the rule fails. Allocate the string first and use `errdefer` to free it on error.